wiki/introduction/setting-up.md

```diff
@@ -111,7 +111,7 @@ You'll need to update a few things under `"quilt_loader"`, see below for a final
       but typically people put a `"homepage"`, a `"sources"`, and a `"issues"` entry with a valid URL here.
    5. Replace `example_mod` with your mod's ID in `"icon"`.
 4. In `"entrypoints"`, replace `com.example.example_mod` with your Maven group and mod ID,
-   and the `ExampleMod` at the end should be the Java class name for your mod. The Java class name usually is your mod's name, written in `UpperCamelCase` (So new words simply start by using a uppercase letter and the initial latter is also upper case)
+   and the `ExampleMod` at the end should be the Java class name for your mod. The Java class name usually is your mod's name, written in `UpperCamelCase` (So new words simply start by using an uppercase letter and the initial latter is also upper case)
```

Comment:
`uppercase latter` is another typo, this sentence is also semantically pretty bad and should just get a rewrite